public/types/index.ts

```diff
@@ -107,12 +108,21 @@ export type EvaluationExperiment =
     judgmentId: string;
   })
 
+export type HybridOptimizerExperiment =
+  (ExperimentBase & {
+    type: "HYBRID_OPTIMIZER";
+    searchConfigurationId: string;
+    judgmentId: string;
+  })
+
 export const printType = (type: string) => {
   switch (type) {
     case "PAIRWISE_COMPARISON":
       return "Comparison";
     case "POINTWISE_EVALUATION":
       return "Evaluation";
+    case "HYBRID_OPTIMIZER":
+      return "HybridOptimizer";
```

Comment:
I wonder if this should have a space?  since it's for human use? `Hybrid Optimizer`...
```suggestion
      return "Hybrid Optimizer";
```

Reply:
ok, let me check if that works I'll push new commit

Reply:
ack, it works, pushed commit to address this
should be good to merge now @epugh